data/json/recipes/recipe_others.json

```diff
@@ -4157,14 +4157,15 @@
   {
     "type": "recipe",
     "result": "neoprene",
-    "category": "CC_CHEM",
-    "subcategory": "CSC_CHEM_OTHER",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
     "skill_used": "cooking",
     "difficulty": 5,
     "time": "25 m",
     "book_learn": [ [ "adv_chemistry", 3 ], [ "textbook_chemistry", 3 ] ],
+    "result_mult": 9,
     "qualities": [ { "id": "CHEM", "level": 2 } ],
-    "tools": [ [ [ "hotplate", 100 ], [ "toolset", 100 ] ] ],
+    "tools": [ [ [ "surface_heat", 100, "LIST" ] ], [ [ "sheet_metal", -1 ] ] ],
     "components": [
```

Comment:
Why is there a sheet_neoprene and a generic neoprene, and if it makes sense to have both does it make sense to turn the components into a requirement?

Reply:
One is a rough patch of neoprene, another one is a purpose-made sheet for making neoprene gear. I've decided to keep both for now since the tailoring overhaul is nowhere close to being done.